dali/kernels/imgproc/resample/params.h

```diff
@@ -37,27 +37,30 @@ inline const char * FilterName(ResamplingFilterType type) {
 
 constexpr int KeepOriginalSize = -1;
 
-inline float DefaultFilterRadius(ResamplingFilterType type, float in_size, float out_size) {
+inline float DefaultFilterRadius(ResamplingFilterType type, bool antialias,
+                                 float in_size, float out_size) {
```

Comment:
Nitpick: here the order is _type,_ **antialias** _in_size, out_size_
In FilterDesc the order is _type, radius,_ **antialias**
I don't know which order is better, but perhaps I'd suggest sticking to one.